mne/preprocessing/ica.py

```diff
@@ -514,6 +516,7 @@ def _fit_raw(self, raw, picks, start, stop, decim, reject, flat, tstep,
 
         # this will make a copy
         if (reject is not None) or (flat is not None):
+            self.reject_ = reject
```

Comment:
if you fit on epochs you should also take it from epochs.reject I think

Reply:
I'm afraid this would be too much magic and would cause confusion in users (especially if the for example they fit ICA on epochs and later evaluate on the continuous dataset, or fit ICA on one set of epochs and want to later evaluate on all).

Reply:
Yes I agree. At least this needs to be discussed.